### PR TITLE
[stable/spring-cloud-data-flow] different MySQL improvements

### PR DESCRIPTION
**Pull request by sebastien-prudhomme**
#### What this PR does / why we need it:

- remove hardcoded image name: use the values from the MySQL subchart
- remove hardcoded database name: use the value that yet exists
- update the version of the MySQL subchart

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/spring-cloud-data-flow/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Toolkit for building data processing pipelines.
 name: spring-cloud-data-flow
-version: 2.0.2
+version: 2.0.3
```
Comment (cpanato) [inline stable/spring-cloud-data-flow/Chart.yaml]: please update to 
```suggestion
version: 2.1.0
```

Comment (sebastien-prudhomme) [reply stable/spring-cloud-data-flow/Chart.yaml]: Done

**Comment by cpanato** [comment]
@sebastien-prudhomme please update your PR with latest master and fix the conflict

**Comment by sebastien-prudhomme** [comment]
> @sebastien-prudhomme please update your PR with latest master and fix the conflict

Done

**Review by cpanato — APPROVED** [review]

**Pull request merged**